docs/technical_reference/costing/electroNP.rst

```diff
@@ -9,51 +9,51 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+   "Hydraulic retention time", ":math:`HRT`", "HRT", "1.3333", ":math:`\text{hr}`"
+   "Reactor sizing cost", ":math:`C_V`", "sizing_cost", "1000", ":math:`\text{USD}_{2020}\text{/m}^3`"
+   "Magnesium chloride cost", ":math:`C_{MgCl2}`", "magnesium_chloride_cost", "0.0786", ":math:`\text{USD}_{2020}\text{/kg}`"
+   "Phosphorus recovery value*", ":math:`C_{RP}`", "phosphorus_recovery_value", "-0.07", ":math:`\text{USD}_{2020}\text{/kg}`"
+
+\* Negative value represents revenue from recovering phosphorus
 
 Costing Method Variables
 ++++++++++++++++++++++++
 
-The following variables are constructed on the unit block (e.g., m.fs.unit.costing) when applying the `cost_electroNP` costing method in the ``watertap_costing_package``:
+The following variables are used on the unit block (e.g., m.fs.unit.costing) when applying the `cost_electroNP` costing method in the ``watertap_costing_package``:
 
 .. csv-table::
-   :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
+   :header: "Description", "Symbol", "Variable Name", "Units"
```

Comment:
Based on feedback from yesterday, there should be a column for Index here instead of Default Value.